packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -115,7 +115,12 @@ abstract class RunCommandBase extends FlutterCommand with DeviceBasedDevelopment
         help: 'Filters out all trace events except those that are specified in '
             'this comma separated list of allowed prefixes.',
         valueHelp: 'foo,bar',
-      );
+      )
+      ..addMultiOption('dart-entrypoint-args',
+        abbr: 'a',
+        help: 'Pass a list of arguments to the Dart entrypoint at application '
```

Comment:
I would connect this more strongly with  `main(List<String> args)`, dart entrypoint is a bit vague. I think Dart calls this the "main function".

Reply:
The Dart entrypoint can be any name, the default is main. I will clarify that it's main() by default though.

Reply:
I guess we might have made the configurable in our embedder? But Dart says it is only `main()`